std/protocols/bus.asm

```diff
@@ -125,10 +125,11 @@ let compute_next_z: expr, expr, expr[], expr, Ext<expr>, Ext<expr>, Ext<expr> ->
 
 /// Convenience function for bus interaction to send columns
 let bus_send: expr, expr[], expr -> () = constr |id, tuple, multiplicity| {
-    bus_interaction(id, tuple, multiplicity, 1);
+    // For bus sends, the multiplicity always equals the latch
```

Comment:
So the send can be made conditional by setting both values to zero, right? And the receive can have `multiplicitiy = -1` but `latch=0`?

Reply:
Ah no it cannot. Understood!